src/pages/Messages/_List.js

```diff
@@ -64,7 +66,8 @@ class MessageList extends Component {
 
 const mapStateToProps = (state, { room }) => ({
   currentUser: getCurrentUser(state),
-  messages: getRoomMessages(state, room)
+  messages: getRoomMessages(state, room),
+  getIsStarred: (message) => isStarred(state, message.id)
 })
```

Comment:
One pattern that has changed since you started writing this feature is standardizing the naming inside reducers. Every "getter" method in the reducers now starts with `get*`.

Upshot is these names should flip:

1. The reducer method should be renamed `getIsStarred`
2. The prop name should be renamed `isStarred`